### PR TITLE
implement appimage workflow [skip ci]

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

File: support/scripts/make-appimage.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  exit 1
+fi
+
+if [ ! -e "src/NetPanzer/Core/main.cpp" ];then
+  echo "You need to be in the netpanzer source root directory when you run this script."
+  exit 1
+fi
+
+export WORKSPACE="/np-src-root"
+export VERSION=${VERSION:-"0.8.1"}
```
Comment (winrid) [inline support/scripts/make-appimage.sh:14]: is this the netpanzer version?

**Comment by andy5995** [comment]
This is almost working correctly.

To build the appimage, one need only have docker installed

In the netpanzer source directory, run `support/scripts/prep-appimage.sh` and it should produce an appimage to the support directory.

You can't run it directly, but if you use `./NetPanzer-0.9.0-RC6-x86_64.AppImage --appimage-mount` that will mount it at "/tmp/.mount_NetPanyKwmi8"

cd to that directory, then from there to `usr/bin` and run `./netpanzer`

**Comment by andy5995** [comment]
I'm gonna merge this because I need to test the artifact upload and it looks like it won't work until the included yml is on master

**Comment by andy5995** [comment]
You don't have to build it now. You can download it from the artifacts section at https://github.com/netpanzer/netpanzer/actions/runs/7429545671#artifacts

Download the zip, unzip it and that will extract the AppImage.

**Comment by andy5995** [comment]
I've found that if you move the AppImage into the source root directory and then execute it, it will just work (It will read the data from the source directory instead of the AppImage)

**Pull request merged**